packages/router/router.coffee

```diff
@@ -39,3 +39,22 @@ FlowRouter.route '/organizations/:_id',
     FlowLayout.render 'layout',
       main: 'organizationDetail'
       params: {"organizationId": params._id}
+
+FlowRouter.route '/myDatasets',
```

Comment:
Hmm, I'd probably put my datasets on the same page as the general dataset view - maybe have a button that filters it.